### PR TITLE
Lime-134, Issue 170: Event missing when removing a vote

### PR DESCRIPTION
**Pull request by KalpitaMandal**
# Description
Stacked pull request on #165
When lenders vote to give an extension a vote can be removed if the lender transfers his shares after voting.

When he votes an event will be emitted and if that vote is then cancelled those shares will be able to vote again leading to the number of votes not matching the one reported by the events.

# Integrations Checklist

- [ ] Have any function signatures changed? If yes, outline below.
- [ ]  Have any features changed or been added? If yes, outline below.
- [x]  Have any events changed or been added? If yes, outline below.
- [ ]  Has all documentation been updated?

# Changelog

- Added an event in `IExtension.sol` (Issue 170: Event missing when removing a vote
Ref: https://github.com/code-423n4/2021-12-sublime-findings/issues/170)
- Emitting the above event in `Extension.sol`

# Event Signature Changes
- `event RebalaneVotes(address indexed oldLender, address indexed newLender, uint256 amount);` (Added)
# Function Signature Changes

# Features

# Events

### CODE REVIEW

File: contracts/interfaces/IExtension.sol [hunk 1]
```diff
@@ -34,6 +34,13 @@ interface IExtension {
      */
     event LenderVoted(address indexed lender, uint256 totalExtensionSupport, uint256 lastVoteTime);
 
+    /**
+     * @notice emited when votes are rebalanced of from and to addresses when pool tokens are transferred
+     * @param oldLender address of user from whom pool tokens are transferred
+     * @param newLender address of user to whom pool tokens are transferred
+     */
+    event RebalaneVotes(address indexed oldLender, address indexed newLender);
```
Comment (prateekreddy) [inline contracts/interfaces/IExtension.sol:42]: number of tokens rebalanced also needs to be added as an arg

File: contracts/Pool/Extension.sol [hunk 2]
```diff
@@ -121,6 +121,7 @@ contract Extension is Initializable, IExtension {
                 extensions[_pool].totalExtensionSupport = extensions[_pool].totalExtensionSupport.add(_amount);
             }
         }
+        emit RebalaneVotes(_from, _to);
```
Comment (prateekreddy) [inline contracts/Pool/Extension.sol:124]: consider  adding amount to the event definition

**Review by prateekreddy — CHANGES_REQUESTED** [review]

File: contracts/Pool/Extension.sol (hunk 2 above)
Comment (KalpitaMandal) [reply contracts/Pool/Extension.sol]: Added the amount as an argument

File: contracts/interfaces/IExtension.sol (hunk 1 above)
Comment (KalpitaMandal) [reply contracts/interfaces/IExtension.sol]: Added the tokens rebalanced in the definition.

**Review by prateekreddy — APPROVED** [review]

**Pull request merged**